lenskit-core/src/main/java/org/lenskit/rerank/RerankingRescoredResult.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * LensKit, an open source recommender systems toolkit.
+ * Copyright 2010-2016 LensKit Contributors.  See CONTRIBUTORS.md.
+ * Work on LensKit has been funded by the National Science Foundation under
+ * grants IIS 05-34939, 08-08692, 08-12148, and 10-17697.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 51
+ * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+package org.lenskit.rerank;
+
+import org.lenskit.results.AbstractResult;
+
+/**
+ * A class to store the original score as well as the objective metric score returned by the
+ * objective function algorithm.
+ *
+ * @see AbstractScoringCandidateItemSelector
+ *
+ * @author Daniel Kluver
+ */
+public class RerankingRescoredResult extends AbstractResult {
```

Comment:
Is there a reason we want to specialize this, instead of using the result of [Results.rescore](http://lenskit.org/master/apidocs/org/lenskit/results/Results.html#rescore-org.lenskit.api.Result-double-)?